shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/data/remote/constant/Endpoints.kt

```diff
@@ -0,0 +1,6 @@
+package co.nimblehq.avishek.phong.kmmic.data.remote.constant
+
+object Endpoints {
```

Comment:
The singular form `Endpoint` would be more consistent with the package names, enums, etc.
```suggestion
object Endpoint {
```

Reply:
solved in c4beaf7